frontend/src/pages/student/Facharbeit/FacharbeitApplicationForm.tsx

```diff
@@ -57,6 +62,21 @@ function FacharbeitApplicationForm() {
     });
   };
 
+  const handleSubmitAdminFacharbeit = () => {
+    const requestBody = JSON.stringify({
```

Comment:
{String topic, String subject, String choosenTeacher, String subjectTeacher}
Das sind die parameternamen die laut schnittstellendoku im backend erwartet werden. die namen musst du in deinem requestBody auch verwenden, sonst weiß das backend ja nicht welcher parameter welcher ist. die reinfolge spielt keine rolle, nur die Namen

Reply:
wenn du ein JS object übergibst, dann ist die notation so: {key: value}

was du jetzt gemacht hast ist die values umbenennen.

kleines Bsp: du hast dein requestBody so definiert: 
requestbody = {topic: topicArea}
wenn du jetzt den wert von topicArea auf 'asdf' setzt, dann sieht dein objekt so aus:
{topicArea: 'asdf'}

aber wenn du dein requestBody so definierst: requestBody = {topic: blablablubb} und du den wert blablablubb auf 'asdf' setzt, dann sieht dein resultierendes objekt genauso aus wie im ersten Bsp. da der name des values ja ersetzt wird.

Heißt zusammengefasst: das objekt {topic: topicArea} und das objekt {topic: blablablubb} sehen genau gleich aus, wenn topicArea und blablablubb den gleichen wert haben.

deshalb schaut wer auch immer das objekt übergeben bekommt nicht auf den namen des values, den du vergibst, sonder auf den namen des key.

zusammengefasst: wenn das backend einen parameter "subject" erwartet, dann musst du den key auch "subject" nennen und nicht den value, da der value ja zum schluss durch einen wert ersetzt wird